containers/Dockerfile.EESSI-build-node-debian12

```diff
@@ -37,8 +37,9 @@ RUN mkdir -p /cvmfs/software.eessi.io
 
 RUN useradd -ms /bin/bash eessi
 
+RUN python3 -m venv /opt/archspec && source /opt/archspec/bin/activate && pip3 install archspec && deactivate && ln -s /opt/archspec/bin/archspec /usr/local/bin/archspec
```

Comment:
You may want to pin the version of archspec, the latest release doesn't work with our scripts